src/components/applications/ApplicationsView.js

```diff
@@ -23,15 +25,21 @@ export const ApplicationsView = ({getApplications, isHR}) => {
                 setApplications(data)
                 selectedApplication && setSelectedApplication(data.filter(application => application.id === selectedApplication.id)[0])
                 setFetching(false)
-            }).catch(() => setFetching(false))
+            }).catch(() => {
+                Swal.fire({
+                    title: "Error",
+                    text: "We weren't able to get this offer's applications! You will be redirected to your offers",
+                    icon: "error"
+                }).then(() => {
+                    setFetchError(true)
+                    setFetching(false)
```

Comment:
Czy nie walnąć tego do funkcji w której przekażesz jednej funkcji, żeby uniknąć sytuacji że ktoś ustawisz obie wartości na true

Reply:
Ten setFetching jest tutaj tak naprawdę zbędny, bo to tak wygląda, że 
```
(fetchError, fetching) match {
  case (true, _) => redirect()
  case (false, true) => kręciołek()
  case (false, false) => aplikacje()
}
```
Jest tu dla zachowania rozumu i godności człowieka, ale można se ustawiać co dusza zapragnie